src/sentry/static/sentry/app/components/stackedBarChart.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import createReactClass from 'create-react-class';
 import PropTypes from 'prop-types';
-import moment from 'moment';
+import moment from 'moment-timezone';
```

Comment:
Do you need to import from 'moment-timezone'? You're not doing anything with `moment.tz`

Reply:
The timezone is probably configured into moment timezone somewhere, and this is just making the usage timezone aware.

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/stores/configStore.jsx#L32

Reply:
`moment()` just displays in UTC

Reply:
Ah, (unrelated to this PR), should we update most of our other `moment` imports too?